### DISCUSSION TITLE
Notification of intention to package for Fedora

### DISCUSSION THREAD

**Discussion by iSaluki** [helix-editor/helix#8383] [General] [answered]
I'm interested in packaging Helix for Fedora, so that users dont need to add a COPR repo to use Helix on their system. 

This process may take a while to complete, and I am notifying you of my intention to begin. If you have any objections, or wish to make any comments about this, then please let me know here or email saluki@fedoraproject.org. Thanks for making an awesome project!

**Comment by dead10ck** [comment]
That's awesome to hear! As a Fedora user myself, that makes me very excited. Please feel free to reach out on Matrix or on here if you need any questions answered.

**Comment by iSaluki** [comment, marked as answer]
I've taken quite a minor role in this process, as blinxen has done all the heavy lifting (so they're the one you'll want to thank when this is done). 

It makes sense for me to end this discussion here unless anyone has any urgent comments, as I won't be personally responsible for this going forward and questions related to Helix packaging for Fedora will be better directed elsewhere.
